src/components/AuthLayout.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Form, NavLink } from 'react-router-dom';
+import PropTypes, { bool } from 'prop-types';
+
+import Register from './Register';
+import Login from './Login';
+
+import './auth-layout.css';
+
+const AuthLayout = (props) => {
+    const isRegistration = props.isRegistration;
+
+    return (
+        <>
```

Comment:
We'll need to se if everyone agrees with this, but mostly we try to avoid using react fragments unless really necessary. I think this could be a section with articles etc within the section. Even if it has no ID

Reply:
Agree. We never use a Fragment unless we have to